communitiesuk/funding-service-design-post-award-data-store
# SMD-769 - Order error cell indexes logically

https://dluhcdigital.atlassian.net/browse/SMD-773

### Change description
_A brief description of the pull request_



### How to test
* Edit a valid spreadsheet and introduce multiple instances of a specific type of error across the same rows and/or columns.
* On `main` branch, upload the spreadsheet and confirm that the cell references as unordered.
* Checkout this branch, re-upload and confirm that the cell references are now ordered.


### Screenshots of UI changes (if applicable)
#### Before
<img width="1067" alt="image" src="https://github.com/communitiesuk/funding-service-design-post-award-data-store/assets/2920760/9703ab86-af84-4dbd-8b0b-a76d88d6a408">


#### After
<img width="1093" alt="image" src="https://github.com/communitiesuk/funding-service-design-post-award-data-store/assets/2920760/8b0ef636-1e9c-4c76-bf62-4a91e6fe2b09">
